static/i18n/en/app.yml

```diff
@@ -218,6 +218,8 @@ manager:
     help: Check on your device to see which apps are already installed
   firmware:
     installed: 'Firmware version {{version}}'
+    titleNano: Ledger Nano S
+    titleBlue: Ledger Blue
```

Comment:
sry didnt understood your question earlier, but if it's not used anywhere we may not put it in translation file

Reply:
yeah, it's not to be translated.

Reply:
>< oops missed the usage. LGTM